src/CommunityToolkit.Maui.Core/Handlers/DrawingView/DrawingViewHandler.android.cs

```diff
@@ -0,0 +1,117 @@
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapClearOnFinish(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.ClearOnFinish = view.ClearOnFinish;
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.LineColor"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapLineColor(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.LineColor = view.LineColor.ToPlatform();
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.LineWidth"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapLineWidth(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.LineWidth = view.LineWidth;
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.MultiLineMode"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapMultiLineMode(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.MultiLineMode = view.MultiLineMode;
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.DrawingLineCompletedCommand"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapDrawingLineCompletedCommand(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.DrawingLineCompletedCommand = view.DrawingLineCompletedCommand;
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.DrawingLineCompletedCommand"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	/// <param name="parameter">Command argument</param>
+	public static void MapDrawingLineCompletedEvent(DrawingViewHandler handler, IDrawingView view, object? parameter)
+	{
+	}
+
+	/// <inheritdoc />
+	protected override void ConnectHandler(MauiDrawingView nativeView)
+	{
+		base.ConnectHandler(nativeView);
+		nativeView.Initialize();
+		VirtualView.Lines.CollectionChanged += Lines_CollectionChanged;
+	}
+
+	/// <inheritdoc />
+	protected override void DisconnectHandler(MauiDrawingView nativeView)
+	{
```

Comment:
Just a heads up, that we have a bug that this method isn't called, see [here](https://github.com/dotnet/maui/issues/3604), so do we want to do a workaround calling it manually (if it's possible) or not?

Reply:
I don't think we need to call it manually. it is a Maui issue and I believe it happens for all controls. For now, we can keep it and wait for the new Maui release. If they don't fix it after GA, we'll need to find a workaround and call it manually.